index.html

```diff
@@ -9222,6 +9225,256 @@ <h3 id="example-2">
+          <div class="note">
+            This implements the pre-delay and the application of the reduction
+            gain.
+          </div>
+          <p>
+            The following algorithm describes the <code>EnvelopeFollower</code>
+            object to be applied to the input signal to produce the gain
+            reduction value. First, a couple variables that persist accross
+            invocation of this algorithm, tied to this particular
+            <code>EnvelopeFollower</code> instance.
+          </p>
+          <ul>
+            <li>Let <var>detector average</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+            <li>Let <var>compression gain</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+          </ul>
+          <p>
+            Now, the actual algorithm to determine the reduction gain, in
+            linear gain value.
+          </p>
+          <ol>
+            <li>Let <var>input signal</var> be the the render quantum of audio
+            to be processed.
+            </li>
+            <li>Let <var>threshold</var>, <var>knee</var> have the value of the
+            <a>AudioParam</a> of the same name, <a href=
+            "#db-to-linear">converted to linear unit</a> sampled at the time of
+            processing of this block (as <a>k-rate</a> parameters).
+            </li>
+            <li>Let <var>ratio</var> have the value of the <code>ratio</code>
+            <a>AudioParam</a>, sampled at the time of processing of this block
+            (as a <a>k-rate</a> parameter).
+            </li>
+            <li>Let <var>attack</var> and <var>release</var> have the value of
+            the <a>AudioParam</a> of the same name, sampled at the time of
+            processing (those are <a>k-rate</a> parameters), mutiplied by the
+            sample-rate of the <a>BaseAudioContext</a> this
+            <a>DynamicsCompressorNode</a> is <a href=
+            "#associated">associated</a> with.
+            </li>
+            <li>For each samples <var>input</var> of <var>input signal</var>,
+            execute the following steps:
+              <ol>
+                <li>Let <var>envelope rate</var> be the result of <a href=
+                "envelope-rate">computing the envelope rate</a>
+                </li>
+                <li>Let <var>attenuation</var> be the value of applying the
+                <a href="compression-curve">compression curve</a> to the
+                absolute value of <var>input</var> .
+                </li>
+                <li>Set <var>change-rate</var> to the difference between <var>
+                  attenuation</var> and <var>detector average</var>. Multiply
+                  <var>change-rate</var> by the <var>envelope rate</var>. Add
+                  <var>change-rate</var> to <var>detector average</var>, and
+                  clamp <var>detector average</var> to a maximum of 1.0.
+                </li>
```

Comment:
I think there's a mistake here. In the algorithm, change-rate is not multiplied by the envelope rate, but by a different variable named `rate`, which in turn is derived from the  attenuation in dB divided by the baked-in parameter `sat_release_frames`.